packages/gatsby/.npmignore

```diff
@@ -10,3 +10,5 @@
 # Gatsby specific
 !gatsby-browser.js
 !gatsby-node.js
+!gatsby-browser.d.ts
+!gatsby-node.d.ts
```

Comment:
Question: Do we want to include these declaration files in the tarball? If yes, then we need to adjust the Gatsby-specific `prepack` script to copy them to `build`, too. Currently they do not end up in the tarball. Or is there another reason why we're adding them to `.npmignore`?

Reply:
> Do we want to include these declaration files in the tarball? 

I was on the fence about this. I opted for including them because they're useful for debugging, but I don't feel strongly about it. What do you think?

> If yes, then we need to adjust the Gatsby-specific prepack script to copy them to build, too.

Oh, good catch. Unless you think it's a terrible idea to add them, I'll fix this.

Reply:
I think it can't hurt to include them. Don't know enough about Gatsby to know how many people would need them but if we're already producing them why not. We have to copy them to `build/types` to be found by the `types` entry point, i think, right?

Reply:
One thing just came to my mind: If, in the tarball, they end up in `build/types`, we could just adjust `tsconfig.plugin.json` to output them to `build/types`, right? Then we don't have to adjust Gatsby's `prepack.ts`.

Reply:
Update: I added them to the prepack script (and a few other places I'd forgotten about), but left them alongside the the source files they describe.

Reply:
Very good point, I didn't think about that. I say if it works for the source file this way, then why not for the declarations as well. Sounds good.